doc/ref_arch/kubernetes/chapters/chapter04.rst

```diff
@@ -32,31 +32,191 @@ virtual machine and an installed Operating System. In order for a Kubernetes Nod
+|``ra2.ch.016``|Physical PCIe |The physical machines on which the Kubernetes |                       |                       |
+|              |slots         |Nodes run **must** be equipped with at least  |                       |                       |
+|              |              |eight (8) Gen3.0 PCIe slots, each with at     |                       |                       |
+|              |              |least eight (8) lanes.                        |                       |                       |
++--------------+--------------+----------------------------------------------+-----------------------+-----------------------+
+|``ra2.ch.017``|Immutable     |Whether physical or virtual machines are      |`req.gen.cnt.02 <./chap|`4.3.1 <../../../ref_im|
+|              |infrastructure|used, the Kubernetes Node **must not** be     |ter02.md#kubernetes-arc|pl/cntt-ri2/chapters/ch|
+|              |              |changed after it is instantiated. New changes |hitecture-require      |apter04.md#installation|
+|              |              |to the Kubernetes Node must be implemented as |ments>`__              |-on-bare-metal-infratru|
+|              |              |new Node instances. This covers any changes   |                       |cture>`__              |
+|              |              |from BIOS through Operating System to running |                       |                       |
+|              |              |processes and all associated configurations.  |                       |                       |
++--------------+--------------+----------------------------------------------+-----------------------+-----------------------+
+|``ra2.ch.018``|NFD           |`Node Feature Discovery <https://kubernetes-si|TBD                    |`4.3.1 <../../../ref_im|
+|              |              |gs.github.io/node-feature-discovery/stable/get|                       |pl/cntt-ri2/chapters/ch|
+|              |              |-started/index.html>`__ **must** be used to ad|                       |apter04.md#installa    |
+|              |              |vertise the detailed software and hardware    |                       |tion-on-bare-metal-infr|
+|              |              |capabilities of each node in the Kubernetes   |                       |atructure>`__          |
+|              |              |Cluster.                                      |                       |                       |
++--------------+--------------+----------------------------------------------+-----------------------+-----------------------+
 
 **Table 4-1:** Node Specifications
 
+Node Operating System
+---------------------
+
+In order for a Host OS to be compliant with this Reference Architecture it must meet the following requirements:
+
++--------------+--------------+----------------------------------------------+-----------------------+-----------------------+
```

Comment:
Please leave a space before and after the borders of the table. This is needed for proper rst rendering.

Reply:
done.